internal/services/loganalytics/log_analytics_cluster_customer_managed_key_resource.go

```diff
@@ -58,6 +60,12 @@ func resourceLogAnalyticsClusterCustomerManagedKey() *pluginsdk.Resource {
 				Required:     true,
 				ValidateFunc: keyVaultValidate.NestedItemIdWithOptionalVersion,
 			},
+
+			"key_rsa_size": {
```

Comment:
Can we split this into its own PR?

Reply:
Because the quota limit of the service split these into different PRs will cost a very long time to run these tests..